### ISSUE TITLE
Support for cropping non-iiif backed resources

### ISSUE THREAD

**Issue by jkeck** [projectblacklight/spotlight#1659]
Once IIIF Cropping is implemented, we'll need a way to handle indexed resources that are not backed by IIIF.

One approach that we can take (that would require little design up-front) is to simply provide a notice that the non-iiif backed item must be uploaded before being cropped.

Another approach (which may take more design and development effort, and we could potentially iterate on after the initial approach) would be to request the image for the user and IIIF-ify it for them.  Not sure what the implications or usability concerns might be around doing this.

**Comment by jkeck** [comment]
@ggeisler feel free to clean any of that up.  Not trying to get into too many details, just trying to make sure we have the papertrail to remember our convo.

**Comment by ggeisler** [comment]
Yep, that accurately reflects our conversation. As a placeholder for the quicker, first approach, below is a mockup of adding a notice when the user submits a non-IIIF image. I think the same notice, placed where it is, would work whether the user was trying to use an exhibit or an uploaded image. 

However, if we think this will be anything more than a short-term solution, we'd probably want to think about this more. One concern I have is that in the near-term at least, the large majority of users using this page are unlikely to know what IIIF even is. So giving them an actionable way to overcome the error is challenging. That would be part of an argument for considering the second approach, despite the more involved effort.

![configuration_-_appearance___map_view_testing_-_online_exhibits__stage_](https://cloud.githubusercontent.com/assets/101482/21333493/6306c4fc-c60d-11e6-8194-c17c255532b3.png)
